.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,4 +17,4 @@ See also #23
 - [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
 - [ ] Affected artifact templates in `packages/cli` were updated
 - [ ] Affected example projects in `examples/*` were updated
-- [ ] Agree to the CLA (Contributor License Agreement) by [clicking and signing](https://cla.strongloop.com/agreements/strongloop/loopback-next)
+- [Sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) - Done once per repository
```

Comment:
Just a suggestion... would it be better to have something like "Sign the CLA if you haven't done so for this repo"?
If someone only cares or knows about `loopback-next` repo, they don't need to know about the "per repository"?

Reply:
Maybe use an emoji, such as: :point_right: [Sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next)

Reply:
:point_right: